app/controllers/facility_reservations_controller.rb

```diff
@@ -141,6 +141,7 @@ def update_admin
 
     @reservation.assign_times_from_params(params[:admin_reservation])
     @reservation.admin_note = params[:admin_reservation][:admin_note]
+    @reservation.expires_mins_before = params[:admin_reservation][:expires_mins_before]
     @reservation.category = params[:admin_reservation][:category]
```

Comment:
If we can use strong params to make this a little cleaner, I would vote for that.

Reply:
It's a also a little weird the stylistic difference from `create_admin`. Maybe there's a reason they're different, but if we can make them more similar, that'd be great.